api/src/test/java/org/openmrs/util/UserByNameComparatorTest.java

```diff
@@ -9,58 +9,110 @@
  */
 package org.openmrs.util;
 
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
-
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.openmrs.Person;
 import org.openmrs.PersonName;
 import org.openmrs.User;
 
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
 /**
  * This test class (should) contain tests for all of the {@link UserByNameComparator} methods.
  */
 public class UserByNameComparatorTest {
-	
+
+	private User user1;
+	private User user2;
+	private User nullUser;
+
+	@Before
+	public void setUp() throws Exception {
+		Person person1 = new Person();
+		person1.addName(new PersonName("givenName", "middleName", "familyName"));
+		user1 = new User(person1);
+		Person person2 = new Person();
+		person2.addName(new PersonName("givenName", "middleNamf", "familyName"));
+		user2 = new User(person2);
+		nullUser = null;
+	}
+
 	/**
 	 * This tests sorting with the {@link UserByNameComparator} given a set of users with
 	 * personNames
-	 * 
-	 * @see UserByNameComparator#compare(User,User)
+	 *
+	 * @see UserByNameComparator#compare(User, User)
 	 */
 	@Test
 	public void compare_shouldSortUsersByPersonNames() {
-		
-		Person person1 = new Person();
-		person1.addName(new PersonName("givenName", "middleName", "familyName"));
-		User user1 = new User(person1);
-		Person person2 = new Person();
-		person2.addName(new PersonName("givenName", "middleNamf", "familyName"));
-		User user2 = new User(person2);
```

Comment:
@ayush2297 are these changes part of the ticket or formatting changes ?

Reply:
these are formatting changes.. i didn't want to repeat code so i moved these declarations at test level and set them up in setup() method and then reused the variables in my tests.

Reply:
oh okay